src/ApolloProvider.tsx

```diff
@@ -61,7 +69,6 @@ export default class ApolloProvider extends Component<ProviderProps, any> {
     // intialize the built in store if none is passed in
     props.client.initStore();
     this.store = props.client.store;
```

Comment:
Shouldn’t the client and store be state instead of class properties? Or better yet, why do we even need them to be class properties? We can just call `initStore` on them when they change.